### PR TITLE
fix traitdefinition keda-scaler 

### PR DESCRIPTION
**Pull request by yangsoon**
fix keda-scaler demo

### CODE REVIEW

File: registry/keda-scaler.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      template: |
+        outputs: kedaScaler: {
+        	apiVersion: "keda.sh/v1alpha1"
+        	kind:       "ScaledObject"
+        	metadata: {
+        		name: context.name
+        	}
+        	spec: {
+        		scaleTargetRef: {
+        			name: context.name
```
Comment (zzxwill) [inline registry/keda-scaler.yaml:24]: No namespace?

File: registry/keda-scaler.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        		}
+        		triggers: [{
+        			type: parameter.type
+        			metadata: {
+        				type:  "Utilization"
+        				value: parameter.value
```
Comment (zzxwill) [inline registry/keda-scaler.yaml:30]: Here are two types, let's make it more clear. You can refer to https://github.com/oam-dev/catalog/blob/70d63115f238b01dd76f3898d6ca6efce4fe66c5/traits/autoscalertrait/api/v1alpha1/autoscaler_types.go#L64-L73. @wonderflow and I discussed it several times.

The reason why you are encouraged to modify it is the spec of keda and HPA are mixed-up, sometime confusing.

File: registry/keda-scaler.yaml (hunk 1 above)
Comment (yangsoon) [reply registry/keda-scaler.yaml:24]: The namespace field is not included here.

https://github.com/kedacore/keda/blob/1f48e663f4c053a3ff3d7d01c4a82f97b5147141/api/v1alpha1/scaledobject_types.go#L64

File: registry/keda-scaler.yaml (hunk 2 above)
Comment (yangsoon) [reply registry/keda-scaler.yaml:30]: got it

**Review by zzxwill — APPROVED** [review]
/lgtm

**Pull request merged**